librosa/core/audio.py

```diff
@@ -532,6 +540,7 @@ def resample(
     scipy.signal.resample
     resampy
     samplerate.converters.resample
+    soxr
```

Comment:
This will only work properly as a cross-ref if we have an intersphinx target for soxr.